runners/samza/src/main/java/org/apache/beam/runners/samza/SamzaRunner.java

```diff
@@ -81,12 +81,12 @@ private SamzaRunner(SamzaPipelineOptions options) {
   }
 
   public PortablePipelineResult runPortablePipeline(RunnerApi.Pipeline pipeline, JobInfo jobInfo) {
-    final String dotGraph = PipelineDotRenderer.toDotString(pipeline);
-    LOG.info("Portable pipeline to run:\n{}", dotGraph);
+    final String jsonGraph = PipelineJsonRenderer.toJsonString(pipeline).toString();
```

Comment:
why are we invoking toString() on a string object ?

Reply:
That change was because I was returning an Optional object. I have now changed the `toJsonString` method to return an `Optional<String>` and changed this call to `PipelineJsonRenderer.toJsonString(pipeline).get();`.